ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py

```diff
@@ -2642,6 +2642,40 @@ def test_jax_numpy_polyadd(
     )
 
 
+# polyder
+@handle_frontend_test(
+    fn_tree="jax.numpy.polyder",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=1,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=1,
+    ),
+    m=st.integers(min_value=0, max_value=10),
+)
+def test_jax_numpy_polyder(
+        *,
+        dtype_and_x,
+        m,
+        test_flags,
+        on_device,
+        fn_tree,
+        frontend,
+):
+    input_dtype, x = dtype_and_x
+    assume("float16" not in input_dtype)
```

Comment:
It's best practice to define the unspported data types of a function using our corresponding decorators instead of filtering the dtypes in the test function. Please have a look at this [section](https://lets-unify.ai/docs/ivy/overview/deep_dive/data_types.html#supported-and-unsupported-data-types) of our docs to make the appropriate modifications.
Other than that, the PR is great 🙂